packages/styles/scss/components/masthead/_masthead.scss

```
@@ -305,6 +315,68 @@ $search-transition-timing: 95ms;
     }
   }
 
+  .#{$prefix}--header__action-left-caret,
+  .#{$prefix}--header__action-right-caret {
+    width: 40px;
+    display: none;
+    position: relative;
+    @media (min-width: 800px) {
+      display: block;
+    }
+  }
+  .#{$prefix}--header__action-left-caret {
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      left: -2px;
+      top: 50%;
+      transform: translateY(-50%);
+      height: 1.5rem;
+      width: 0.0625rem;
+      background-color: #dcdcdc;
+      &:focus,
+      &:active {
+        display: none;
+      }
+    }
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      right: -10px;
+      top: -2px;
+      height: 48px;
+      width: 8px;
+      background: linear-gradient(to right, $ui-background, transparent);
+      z-index: 1;
+    }
+  }
+  .#{$prefix}--header__action-right-caret {
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      right: -2px;
+      top: 50%;
+      transform: translateY(-50%);
+      height: 1.5rem;
+      width: 0.0625rem;
```

Continuing on @jeffchew's `rem` point:

```suggestion
      height: rem(48px);
      width: rem(8px);
```